Added new Client to "Clients" page

appsynergydev opened the pull request:
Hi, I have a Jellyfin client I would like to add to the clients page. 

Thank you.


<img width="1214" alt="Screenshot 2024-02-04 at 19 14 45" src="https://github.com/jellyfin/jellyfin.org/assets/158848199/9b71b0ab-0a6c-4377-a10c-92d408c4dbec">

nielsvanvelzen commented on src/data/clients.ts:
```diff
@@ -637,6 +637,30 @@ export const Clients: Array<Client> = [
+  {
+    id: 'phyn',
+    name: 'Phyn',
+    description: 'Multiplatform video and audio playback client for Jellyfin with Jellyseerr support. Available on iOS, iPadOS and macOS. Android coming soon...',
```
- We do not advertise piracy tools (Jellyseerr) from out documentation
- The "Android coming soon" part is not worth mentioning here

appsynergydev replied:
thanks for the feedback. I have fixed it

nielsvanvelzen commented on src/data/clients.ts:
```diff
@@ -637,6 +637,30 @@ export const Clients: Array<Client> = [
     ],
     recommended: true
   },
+  {
```
New entries should be added at the end.

appsynergydev replied:
Thank for the feedback. I have moved it to the end.

nielsvanvelzen commented:
Thanks, the changes look fine now. We're discussing the inclusion of this client internally which will take a bit longer.

appsynergydev commented:
No worries. Thank you

nielsvanvelzen commented:
After some discussion with the other team members (including the core team ) we decided to not include this client. Jellyfin is a FOSS product and adding a proprietary closed-source client with a subscription model does not fit our philosophy.

Some other concerns that were brought up are:
- It's unknown who the actual author is, it feels like they are hiding their identity
- Author made a second GitHub account for this specific pull request which is shady
- The app promotes piracy

I will close the pull request.

appsynergydev commented:
Thank you for the feedback.

I understand your concerns.

From what I gather, the defining reason for this rejection was because of the subscription model. However, the core functionality of the app is totally free. The subscription is just for add-ons which are totally optional. You also do have a subscription based client on your client list so I don't really understand the rationale.

As for your other concerns, those can be easily rectified. The new account for one was because the AppsynergyTS (the company) wanted everything done in their name for what I assume is IP protection reasons but they didn't have a Github account. 

I guess my question is, do I resubmit another request after addressing your concerns?

nielsvanvelzen commented:
The main reason is that the app is closed source and encourages piracy due to inclusion of the jellyseerr support.

> The new account for one was because the AppsynergyTS (the company) wanted everything done in their name for what I assume is IP protection reasons but they didn't have a Github account.

Can you explain this in more detail? Are you being paid to develop this app for a company?

appsynergydev commented:
> The main reason is that the app is closed source and encourages piracy due to inclusion of the jellyseerr support.

I undersand. 

> Can you explain this in more detail? Are you being paid to develop this app for a company?

No, not paid. I was only asked to develop the app landing page and then make the changes in this PR.  However, I will ask the other dev to reach out instead if that will clarify the situation.

Thank you for your time.
